Gems/ROS2/Code/Include/ROS2/Sensor/ROS2SensorComponentBase.h

```diff
@@ -109,14 +109,15 @@ namespace ROS2
         //! Returns a complete namespace for this sensor topics and frame ids.
         [[nodiscard]] AZStd::string GetNamespace() const
         {
-            auto* ros2Frame = Utils::GetGameOrEditorComponent<ROS2FrameComponent>(GetEntity());
+            auto* ros2Frame = GetEntity()->template FindComponent<ROS2FrameComponent>();
+
             return ros2Frame->GetNamespace();
         }
 
         //! Returns this sensor frame ID. The ID contains namespace.
         [[nodiscard]] AZStd::string GetFrameID() const
         {
-            auto* ros2Frame = Utils::GetGameOrEditorComponent<ROS2FrameComponent>(GetEntity());
+            auto* ros2Frame = GetEntity()->template FindComponent<ROS2FrameComponent>();
```

Comment:
```suggestion
            auto* ros2Frame = GetEntity()->FindComponent<ROS2FrameComponent>();
```

Reply:
Take a look if this is what was meant here

Reply:
This is required here as per ISO C++
```

ISO C++03 14.2/4:

    When the name of a member template specialization appears after . or -> in a postfix-expression, or after nested-name-specifier in a qualified-id, and the postfix-expression or qualified-id explicitly depends on a template-parameter (14.6.2), the member template name must be prefixed by the keyword template. Otherwise the name is assumed to name a non-template.
```